curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-localstorage-by-building-a-todo-app/64ec9282cd547785258cecf2.md

```diff
@@ -20,13 +20,13 @@ assert.match(code, /taskData\.forEach\(\s*\(\s*\{\s*(?:id\s*,\s*title\s*,\s*date
 You should use arrow syntax and attach `innerHTML` to `tasksContainer`.
 
 ```js
-assert.match(code, /taskData\.forEach\(\(\{.*\}\)\s*=>\s*(\(\s*tasksContainer\.innerHTML\s*\))?/)
+assert.match(code, /taskData\.forEach\(\(\{.*\}\)\s*=>\s*(\s*\{\(\s*tasksContainer\.innerHTML\s*\)\s*\})?/)
 ```
 
 You should use addition assignment to set the `innerHTML` of `tasksContainer` to an empty pair of backticks.
 
 ```js
-assert.match(code, /taskData\.forEach\(\(\{.*\}\)\s*=>\s*(\s*\(?tasksContainer\.innerHTML\s*\+=\s*`\s*`\)?)/)
+assert.match(code, /taskData\.forEach\(\(\{.*\}\)\s*=>\s*\{\s*(\s*\(?tasksContainer\.innerHTML\s*\+=\s*`\s*`\)?\s*\})/)
```

Comment:
Let's update this test as well to allow for both valid solutions 
https://github.com/freeCodeCamp/freeCodeCamp/pull/52988#issuecomment-1879834476

Reply:
Make explicit return also possible .
Please let me know if any further changes needed

Reply:
LGTM 👍